packages/components-native/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Octopusapp Inc.
```

Comment:
Year should be updated, however they all should be so this one can be done and we can update the other ones in another PR